### PR TITLE
Setup F# project

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

**Comment by ianlovell** [comment]
:shipit:

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (jebbinSCL) [inline .gitattributes]: Where did this .gitattributes file come from?

Comment (lclarksl) [inline .gitattributes]: Visual Studio maybe?

Comment (jebbinSCL) [inline .gitattributes]: Hmm, not sure what to go with. You'll definitely need to add the following:
`*.sh eol=lf` for the bash files. 
As for the rest of the stuff, it seems reasonable, but leaves out the fsharp files? E.g. .fs, .fsx, .fsproj.

Defintely add the .sh line, but I'm not sure what would be required for the F# files, if anything

Comment (lclarksl) [inline .gitattributes]: I'm setting up concourse next pull request so I'll get rid of all this and replace it with what we actually need

**Pull request merged**